test/test_utils.js

```diff
@@ -320,6 +320,33 @@ describe("Testing utils functions", function() {
         });
     });
 
+    describe("extract_id_dob", function() {
+        it("valid dates extracted", function() {
+            assert.deepEqual(utils.extract_id_dob("8104267805280"),
+                moment("1981-04-26").format("YYYY-MM-DD"));
+            assert.deepEqual(utils.extract_id_dob("8202017805280"),
+                moment("1982-02-01").format("YYYY-MM-DD"));
+            // assert.deepEqual(utils.extract_id_dob("5409237805280"),
```

Comment:
You can remove this. If we have 88 year old mothers in the system I think we'll be on TV!